src/tcp_orientation_calibration.cpp

```diff
@@ -0,0 +1,182 @@
+#include <tool_point_calibration/tool_point_calibration.h>
+#include <tf/transform_listener.h>
+#include <ros/ros.h>
+#include <tf_conversions/tf_eigen.h>
+#include<Eigen/Dense>
+using namespace Eigen;
+using Eigen::MatrixXd;
+
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "console_tool_calibration");
```

Comment:
it is the same name as the original node. I changed it the node you suggested 

> tool_orientation_calibration

Reply:
The downside is that it can be confusing in the event that this and the original node are started without remapping the node name. As these really are different nodes, this can be confusing.

Reply:
What do you think about the name `tool_orientation_calibration`? I suggested this name solely as it follows the same structure as the original node name and required the least amount of changes

Reply:
I totally agree with you, it follows the same structure of the main one